docs/source/workflow_inputs.rst

```diff
@@ -4628,6 +4630,78 @@ If not entered, the simulation will not include batteries.
 
   A battery in a home without PV or charging/discharging schedules is assumed to operate as backup and is not modeled.
 
+.. _hpxml_vehicles:
+
+HPXML Vehicles
+**************
+
+A single battery electric vehicle can be entered in ``/HPXML/Building/BuildingDetails/Systems/Vehicles/Vehicle``.
+All other vehicle types are currently not modeled.
+This provides detailed modeling of electric vehicles (batteries and charging/discharging) as an alternative to the simple EV charging in :ref:`plug_loads`.
+If not entered, the simulation will not include a detailed electric vehicle model.
+
+  ===============================================================================================  ======  =========  =======================  ========  ===========  ======================================
+  Element                                                                                          Type    Units      Constraints              Required  Default      Notes
+  ===============================================================================================  ======  =========  =======================  ========  ===========  ======================================
+  ``SystemIdentifier``                                                                             id                                          Yes                    Unique identifier
+  ``MilesDrivenPerYear``                                                                           double  miles      >= 0                     No        10,900       Number of miles driven per year
```

Comment:
Put elements in the correct order per the HPXML schema. The `MilesDrivenPerYear` element comes after the `VehicleType` element for example.